### DISCUSSION TITLE
为什么我的绳网要降级！

### DISCUSSION THREAD

**Discussion by fairy-B** [share121/inter-knot#1991] [其他]
![image](https://github.com/user-attachments/assets/5e047464-4ad0-45eb-ac0a-1371e8dac8ca)
这是目前的情况，我降级到了1.8.0才能发的贴
![image](https://github.com/user-attachments/assets/857bcf1b-59d2-48db-a290-c250b6384246)
最新版本是1.10.0，但是不能使用
![image](https://github.com/user-attachments/assets/5e5b4046-5f38-45b6-9e59-5a16affc207f)
这是历史更新记录

**Comment by Dave-12138** [comment]
GitHub pages套了个PWA缓存之后就变成这个上古缓存版本了
建议用[绳网.top](https://www.xn--5o0anh.top/)
